index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="box header">
+            <span>My Logo</span>
+            <span><img src="img/Enabled.png"></span>
+        </div>
+        <div class="box content">
+            <div class="title">
+            I’m Mariane 
+            Glad to see 
+            you!
+            </div>
+            <br>
+            <div class="paragraph">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                   If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
```

Comment:
- [ ] I suggest using `<p class-"paragraph">` tag instead of using `<div>`  tag, 👌
> This way will be semantically correct and the screen readers and other technologies will thank you 💯